install/0000_50_cluster-autoscaler-operator_01_clusterautoscaler.crd.yaml

```diff
@@ -1,8 +1,7 @@
-# TODO(bison): Add schema.
----
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  creationTimestamp: null
```

Comment:
Why do we need this?

Reply:
Generated automatically

Reply:
The CRDs are generated now, by operator-sdk. It adds that bit. We don't need it, but it also doesn't hurt anything.